From: Breno Leitao <leitao@debian.org>
Subject: failcmd: make failcmd.sh executable
Date: Mon, 29 Jul 2024 01:52:11 -0700

Change the file permissions of tools/testing/fault-injection/failcmd.sh to
allow execution.  This ensures the script can be run directly without
explicitly invoking a shell.

Link: https://lkml.kernel.org/r/20240729085215.3403417-1-leitao@debian.org
Signed-off-by: Breno Leitao <leitao@debian.org>
Reviewed-by: Akinobu Mita <akinobu.mita@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 0 files changed

diff --git a/tools/testing/fault-injection/failcmd.sh a/tools/testing/fault-injection/failcmd.sh
old mode 100664
new mode 100755
_
